client/test/unit/components/bhBreadCrumb.spec.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * Created by Dedrick Kitamuka on 23/06/2016.
+ */
+describe('component : bhBreadCrumb', function () {
+    var component, scope, componentController, breadCrumb;
+
+    beforeEach(() => {
+        module('pascalprecht.translate');
+        module('ngStorage');
+        module('angularMoment');
+        module('bhima.services');
+        module('bhima.components');
+    });
+
+    beforeEach(inject(function ($rootScope, _$componentController_){
+        scope = $rootScope.$new();
+        componentController = _$componentController_;
+        breadCrumb = {
```

Comment:
This initialization doesn't need to be in the `beforeEach()` call.  It would be faster to take it outside of the function.  Something like this:
```js
// breadcrumb definition
const breadCrumb = { /* ... */ };

// initialize $injects
beforeEach(inject(($rootScope, _$componentController_) => {
  scope = $rootScope.$new();
  componentController = _$componentController_;
 }));
```